ctapipe/utils/datasets.py

```diff
@@ -112,15 +116,15 @@ def find_in_path(filename, searchpath):
 
     """
 
-    for directory in get_searchpath_dirs(searchpath):
+    for directory in get_searchpath_dirs(searchpath, url):
```

Comment:
`url=url`

Reply:
no problem to add this, why is it needed? that function requires now only 2 args

Reply:
Not needed. Better style. E.g. safe against adding a third as a second argument, like you did before changing to add it at the end ;)

Always prefer kwargs when possible.